src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private int supply;
+    private int buy;
+    private String[] split;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't  create a file", e);
+        }
+        File fileName = new File(toFileName);
+        try {
+            fileName.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't  create a file, e");
+        }
```

Comment:
We dont need to create these files, they are already exist, so lets remove these logic

Reply:
+